packages/search-ui-app-search-connector/src/requestAdapters.js

```diff
@@ -16,16 +16,21 @@ function rollup(f) {
   }));
 
   return {
-    [f.type || "any"]: values
+    [f.type || "all"]: values
   };
 }
 
 function adaptFilters(filters) {
   if (!filters || filters.length === 0) return {};
-  const all = filters.map(rollup);
-  return {
-    all
-  };
+
+  return ["all", "any", "none"].reduce((acc, topType) => {
```

Comment:
We should have tests that cover this logic. For instance, I didn't see anything that tests the "none" case, or a case for values not inside of this list.